eq-author/src/components/Forms/Input/Input.test.js

```diff
@@ -80,4 +80,47 @@ describe("components/Forms/Input", () => {
       );
     });
   });
+
+  describe("Radio", () => {
+    let wrapper;
+
+    beforeEach(() => {
+      wrapper = mount(
+        <Input
+          id="my-radio-button"
+          variant="radioBox"
+          type="radio"
+          onChange={handleChange}
+          defaultChecked={false}
+        />
+      );
+    });
+
+    it("should render a radio button", () => {
+      expect(wrapper.find("input").getElement().props.type).toEqual("radio");
+    });
+
+    it("should pass `defaultChecked` prop to component", () => {
+      wrapper = mount(<Input type="radio" defaultChecked />);
+      expect(wrapper.find("input").getElement().props.defaultChecked).toEqual(
+        true
+      );
```

Comment:
The same as before regarding readability. 

```suggestion
      const inputElement = wrapper.find("input").getElement();
      expect(inputElement.props.defaultChecked).toEqual(true);
```

If you're curious, this is a funny and interesting read about testing booleans in Jest (and JS as a whole) - https://stackoverflow.com/questions/32615713/tobetrue-vs-tobetruthy-vs-tobetrue

JS be cray cray.